.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,21 @@
+name: Feature request
+description: Suggest an idea for this project.
+body:
+  - type: textarea
```

Comment:
Should be marked as `required`

Reply:
> https://github.com/BOINC/boinc/pull/5944#discussion_r1878880787

@AenBleidd, I've committed [`commits/15752cec4889f770caf2c80ab1155b7f6e4865f3#diff-c6b098646bf32c644234bec14c2675ea2a3d9c2dc2df450a25aa0e7ff02323cdR9-R21`](https://github.com/BOINC/boinc/pull/5944/commits/15752cec4889f770caf2c80ab1155b7f6e4865f3#diff-c6b098646bf32c644234bec14c2675ea2a3d9c2dc2df450a25aa0e7ff02323cdR9-R21:~:text=required:%20false-,required:%20true,%2D%20type:%20textarea,-%2D%20type:%20textarea&text=%2D%20type:%20textarea-,%2D%20type:%20textarea,attributes:,-attributes:&text=attributes:-,attributes:,Describe%20the%20solution%20you'd%20like,-label:%20Describe%20the&text=solution%20you'd%20like-,label:%20Describe%20the%20solution%20you'd,you%20want%20to%20happen.%20%2D%2D%3E%22,-placeholder:%20%22%3C!%2D%2D%20A&text=to%20happen.%20%2D%2D%3E%22-,placeholder:%20%22%3C!%2D%2D%20A%20clear%20and,validations:,-validations:&text=validations:-,validations:,required:%20false,-required:%20true&text=required:%20false-,required:%20true,%2D%20type:%20textarea,-%2D%20type:%20textarea&text=%2D%20type:%20textarea-,%2D%20type:%20textarea,attributes:,-attributes:&text=attributes:-,attributes:,label:%20Additional%20context,-label:%20Additional%20context&text=label:%20Additional%20context-,label:%20Additional%20context,or%20features%20you've%20considered.%20%2D%2D%3E%22,-placeholder:%20%22%3C!%2D%2D%20A&text=you've%20considered.%20%2D%2D%3E%22-,placeholder:%20%22%3C!%2D%2D%20A%20clear%20and,validations:,-validations:&text=validations:-,validations:,required:%20false,-required:%20true&text=required:%20false-,required:%20true,-Footer).

Reply:
I see your commits. No need to link them to your replies.